user/views/student.py

```diff
@@ -212,8 +212,6 @@ def put(self, request, *args, **kwargs) -> HttpResponse:
 
 
 class StudentAPIView(LoginRequiredMixin, APIView):
-    # returns permission denied HTTP message rather than redirect to login
-    raise_exception = True
```

Comment:
We will ultimately want to return a permission denied message instead of redirecting to login. The redirect to login won't work now that we have SPA that will be served separately.

As mentioned in your comment, this is dead code at the moment and it seems fine to remove it. Let's open a issue to track this.

Reply:
Good thinking with the redirect issue caused by the SPA. I'll open an issue on creating an appropriate 403.